content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -61,9 +68,13 @@ cannot set `priorityClassName` in new Pods.
 
 ## How to disable preemption
 
+**Note**: Critical Pods in Kubernetes 1.11 and later rely on the scheduler preemption to be
```

Comment:
IIUC this isn't completely accurate -- it should say "In Kubernetes 1.11, critical pods (except DaemonSet pods, which are still scheduled by the DaemonSet controller) rely on scheduler preemption to be scheduled when a cluster is under resource pressure. For this reason, ..."

Reply:
Changed the text as you suggested. PTAL.